### DISCUSSION TITLE
PCAP: Attempt 1: No data available for the requested dates

### DISCUSSION THREAD

**Discussion by DavidKorpiewski** [Security-Onion-Solutions/securityonion#9803] [General]
I'm trying to do a PCAP to see if my SO is even reading data off of the second port which is in promiscuous mode.  However, all of my PCAPs fail with this error:  Attempt 1: No data available for the requested dates
They fail whenever the start time is hit.  So if I set it for 5 minutes in the future, when that time comes up it will fail with this error. If I set the start time in the past, say 2 minutes ago and then say to have it run for 3 more minutes it will fail right away. 
Any idea how to troubleshoot this?
Thank you very much
David
![image](https://user-images.githubusercontent.com/33209406/219645029-17bd0de9-3a44-4d80-b600-44b2a88726f0.png)

![image](https://user-images.githubusercontent.com/33209406/219645212-888fc178-0b75-44c9-939f-3605fe81eb80.png)

![image](https://user-images.githubusercontent.com/33209406/219645294-0b3c1b4d-ed4c-4ae4-88f5-2ca23d8e87f6.png)

**Comment by cm-ops** [comment]
How long is your PCAP retention? You can see that in Grafana.
